src/hooks/useLiveSpecsExt.ts

```diff
@@ -58,9 +58,7 @@ function useLiveSpecsExt(
 
                       return query
                           .eq('spec_type', specType)
-                          .or(
-                              `id.in.(${draftArray}),last_pub_id.in.(${draftArray})`
-                          );
+                          .or(`id.in.(${draftArray})`);
```

Comment:
We no longer need to query by the `last_pub_id` as we were previously. Removing it so we only every query by `id` to make sure this hook is used for specific specs.